src/app/components/get-help/get-help.component.ts

```diff
@@ -23,7 +23,7 @@ export class GetHelpComponent implements OnInit {
     this.userDetails$ = this.store.pipe(select(fromRoot.getUserDetails));
     this.userDetails$.subscribe((userDetail: UserDetails) => {
       if (userDetail && userDetail.userInfo) {
-        this.caseManager = userDetail.userInfo.roles && userDetail.userInfo.roles.includes('pui-case-manager');
+        this.caseManager = userDetail.userInfo?.roles && userDetail.userInfo?.roles.includes('pui-case-manager');
```

Comment:
It's messy to use a combination of tests and optional chaining, and if you are going to put optional chaining in, do it everywhere on the line 
this should be 
this.caseManager = userDetail?.userInfo?.roles?.includes('pui-case-manager');